libs/transactions/src/lib/transactions-journey.component.ts

```diff
@@ -0,0 +1,17 @@
+import { AfterContentChecked, Component, ContentChild } from '@angular/core';
+import { TransactionAdditionalDetailsTemplateDirective } from './directives/transaction-additional-details.directive';
+import { TransactionsJourneyConfiguration } from './services/transactions-journey-config.service';
+
+@Component({
+  selector: 'bb-transactions-journey',
+  templateUrl: 'transactions-journey.component.html',
+})
+export class TransactionsJourneyComponent implements AfterContentChecked {
```

Comment:
It seems currently it is not always common that a journey exports its main journey component, or even that it has one. In this proposal it would be a pre-requisite in order to enable providing templates from the outside